docusaurus/i18n/de/docusaurus-plugin-content-docs/current/advanced/palworld-directory.md

```diff
@@ -0,0 +1,28 @@
+---
+sidebar_position: 1
+---
+
+# Palworld-Verzeichnis
+
+Alles was mit den Palworld-Dateien zusammenhängt, befindet sich im Ordner `/palworld` innerhalb des Containers.
+
+## Ordnerstruktur
+
+![Ordnerstruktur](../assets/folder_structure.jpg)
+
+| Ordner                       | Verwendung                                                             |
+|------------------------------|------------------------------------------------------------------------|
+| palworld                     | Stammordner mit allen Dateien des Palworld-Servers                     |
+| backups                      | Ordner, in dem alle Backups des Befehls `backup` gespeichert werden    |
```

Comment:
Reicht es nicht zu sagen

"Der Ordner, in dem alle Backups gespeichert werden" ?